app/js/app/controllers/LoginCtrl.js

```diff
@@ -36,12 +36,19 @@
             $scope.loginData = {};
```

Comment:
loginData and registerData can be replaced by one variable when you using goTo function. Keep in mind to clear it after each invoke of that method.